bait-files/src/main/java/com/google/sps/data/Keys.java

```diff
@@ -0,0 +1,11 @@
+package com.google.sps.data;
+
+public class Keys {
+  public static final String USER_ENTITY = "User";
+  public static final String USERNAME_ENTITY_PROPERTY = "username";
+  public static final String TIMESTAMP_ENTITY_PROPERTY = "timestamp";
+  public static final String ID_ENTITY_PROPERTY = "id";
+  public static final String FIRST_NAME_ENTITY_PROPERTY = "first-name";
+  public static final String LAST_NAME_ENTITY_PROPERTY = "last-name";
+  public static final String IS_ADMIN_ENTITY_PROPERTY = "is_admin";
+}
```

Comment:
It doesn't look like you're actually using this class anywhere yet... maybe remove it from this PR and add it in again when you're actually going to use it =)

Reply:
Removed!